data/cpu_data.csv

```diff
@@ -0,0 +1,31 @@
+name,cores,threads,release,multi-thread-rating,single-thread-rating
+"AMD EPYC 7401P",24,48,2017,28282,1698
+"AMD Opteron 6338P",12,12,2014,,
+"AMD Ryzen 7 1700X",8,16,2017,15691,2979
+"AMD Ryzen 7 3700X",8,16,2019,22674,2664
+"AMD Ryzen 7 PRO 1700X",8,16,2017,15695,2148
+"AMD Ryzen 9 3900",12,24,2019,30663,2607
+"AMD Ryzen Threadripper 2950X",16,32,2018,29370,2461
```

Comment:
Where did you get this data from :) ?

There're a few very biased websites such as userbenchmark.com, which I don't trust to provide proper data.

Reply:
I know about the userbenchmark bias :D I got everything from passmark as stated in the /help section https://www.cpubenchmark.net/cpu_list.php